$MirOS: ports/www/lynx/patches/patch-lynx_cfg,v 1.14 2010/12/23 20:25:31 tg Exp $

	mostly site policy and compatibility, usability;
	some fix spelling

--- lynx.cfg.orig	Mon Feb 13 02:45:53 2012
+++ lynx.cfg	Sat Mar  3 18:20:21 2012
@@ -1,7 +1,8 @@
+# $MirSecuCron$
+# $MirOS: src/gnu/usr.bin/lynx/lynx.cfg,v 1.33 2012/02/20 02:54:31 tg Exp $
 # $LynxId: lynx.cfg,v 1.215 2012/01/31 23:48:09 tom Exp $
 # lynx.cfg file.
-# The default placement for this file is /usr/local/lib/lynx.cfg (Unix)
-#                                     or Lynx_Dir:lynx.cfg (VMS)
+# The default placement for this file is /etc/lynx.cfg
 #
 # $Format: "#PRCS LYNX_VERSION \"$ProjectVersion$\""$
 #PRCS LYNX_VERSION "2.8.8dev.12"
@@ -12,7 +13,7 @@
 # Definition pairs are of the form  VARIABLE:DEFINITION
 # NO spaces are allowed between the pair items.
 #
-#  If you do not have write access to /usr/local/lib you may change
+#  If you do not have write access to /etc you may change
 #  the default location of this file in the userdefs.h file and recompile,
 #  or specify its location on the command line with the "-cfg"
 #  command line option.
@@ -90,12 +91,12 @@
 #
 # Normally we expect you will connect to a remote site, e.g., the Lynx starting
 # site:
-STARTFILE:http://lynx.isc.org/
+#STARTFILE:http://lynx.isc.org/
 #
 # As an alternative, you may want to use a local URL.  A good choice for this is
 # the user's home directory:
 .ex
-#STARTFILE:file://localhost/~/
+STARTFILE:file://localhost/~/
 #
 # Your choice of STARTFILE should reflect your site's needs, and be a URL that
 # you can connect to reliably.  Otherwise users will become confused and think
@@ -436,7 +437,7 @@ DEFAULT_INDEX_FILE:http://scout.wisc.edu
 # Find RFC 1345 at
 .url http://tools.ietf.org/html/rfc1345
 #
-#CHARACTER_SET:iso-8859-1
+CHARACTER_SET:utf-8
 
 .h2 LOCALE_CHARSET
 # LOCALE_CHARSET overrides CHARACTER_SET if true, using the current locale to
@@ -461,7 +462,7 @@ DEFAULT_INDEX_FILE:http://scout.wisc.edu
 # This option currently implements only the equating of ISO-8859-1 and Windows
 # 1252.
 #
-#HTML5_CHARSETS:FALSE
+HTML5_CHARSETS:TRUE
 
 .h2 ASSUME_CHARSET
 # ASSUME_CHARSET changes the handling of documents which do not
@@ -475,7 +476,7 @@ DEFAULT_INDEX_FILE:http://scout.wisc.edu
 # ASSUME_CHARSET can also be changed via the 'o'ptions menu but will
 # not be saved as permanent value in user's .lynxrc file to avoid more chaos.
 #
-#ASSUME_CHARSET:iso-8859-1
+ASSUME_CHARSET:utf-8
 
 .h2 ASSUMED_DOC_CHARSET_CHOICE
 .h2 DISPLAY_CHARSET_CHOICE
@@ -489,7 +490,7 @@ DEFAULT_INDEX_FILE:http://scout.wisc.edu
 #
 # where
 #
-#	'string' is either the MIME name of charset or it's full name (listed
+#	'string' is either the MIME name of charset or its full name (listed
 #		either in the left or in the right column of table of
 #		recognized charsets), case-insensitive - e.g.  'Koi8-R' or
 #		'Cyrillic (KOI8-R)' (both without quotes),
@@ -596,7 +597,7 @@ DEFAULT_INDEX_FILE:http://scout.wisc.edu
 # the 'o'ptions menu and save that preference in their RC file.
 # This may be a comma-separated list of languages in decreasing preference.
 #
-#PREFERRED_LANGUAGE:en
+#PREFERRED_LANGUAGE:
 
 .h2 PREFERRED_CHARSET
 # PREFERRED_CHARSET specifies the character set in MIME notation (e.g.,
@@ -614,7 +615,7 @@ DEFAULT_INDEX_FILE:http://scout.wisc.edu
 # the sending of an unacceptable response is also allowed.  See RFC 2068
 .url http://tools.ietf.org/html/rfc2068
 #
-#PREFERRED_CHARSET:
+PREFERRED_CHARSET:utf-8
 
 .h2 CHARSETS_DIRECTORY
 # CHARSETS_DIRECTORY specifies the directory with the fonts (glyph data)
@@ -689,21 +690,21 @@ DEFAULT_INDEX_FILE:http://scout.wisc.edu
 # Set the threshold # of lines Lynx must render before it
 # redraws the screen in PARTIAL mode.  Anything < 0 implies
 # use of the screen size.
-#PARTIAL_THRES:-1
+PARTIAL_THRES:3
 
 .h2 SHOW_KB_RATE
 # While getting large files, Lynx shows the approximate rate of transfer.
-# Set this to change the units shown.  "Kilobytes" denotes 1024 bytes:
+# Set this to change the units shown.
 #	NONE to disable the display of transfer rate altogether.
-#	TRUE or KB for Kilobytes/second.
+#	TRUE or KB for Kibibytes/second.
 #	FALSE or BYTES for bytes/second.
-#	KB,ETA to show Kilobytes/second with estimated completion time.
-#	BYTES,ETA to show BYTES/second with estimated completion time.
+#	KB,ETA to show Kibibytes/second with estimated completion time.
+#	BYTES,ETA to show bytes/second with estimated completion time.
 # Note that the "ETA" values are available if USE_READPROGRESS was defined.
-#SHOW_KB_RATE:TRUE
+SHOW_KB_RATE:BYTES,ETA
 
 .h2 SHOW_KB_NAME
-# Set the abbreviation for Kilobytes (1024).
+# Set the abbreviation for Kibibytes (1024 Bytes).
 # Quoting from
 .url http://www.romulus2.com/articles/guides/misc/bitsbytes.shtml
 # In December 1998, the International Electrotechnical Commission (IEC)
@@ -713,9 +714,6 @@ DEFAULT_INDEX_FILE:http://scout.wisc.edu
 # metric prefixes and adding the first two letters of the word "binary".  Thus,
 # for instance, instead of Kilobyte (KB) or Gigabyte (GB), the new terms would
 # be kibibyte (KiB) or gibibyte (GiB).
-#
-# If you prefer using the conventional (and more common) "KB", modify this
-# setting.
 #SHOW_KB_NAME:KiB
 
 .h1 Timeouts
@@ -759,7 +757,7 @@ DEFAULT_INDEX_FILE:http://scout.wisc.edu
 #REPLAYSECS:0
 
 .h1 Appearance
-# These settings control the appearance of Lynx's screen and the way
+# These settings control the appearance of Lynx' screen and the way
 # Lynx renders some tags.
 
 .h2 USE_SELECT_POPUPS
@@ -841,15 +839,15 @@ DEFAULT_INDEX_FILE:http://scout.wisc.edu
 # the least recently displayed documents will be freed until one or the other
 # value is no longer exceeded.  The default value is defined in userdefs.h.
 #
-# The Unix and VMS (but not VAXC) implementations use the C library malloc's
-# and calloc's for memory allocation, but procedures for taking the actual
+# The Unix and VMS (but not VAXC) implementations use the C library malloc
+# and calloc for memory allocation, but procedures for taking the actual
 # amount of cache into account still need to be developed.  They use only
 # the DEFAULT_CACHE_SIZE value, and that specifies the absolute maximum
 # number of documents to cache (rather than the maximum number only if
 # DEFAULT_VIRTUAL_MEMORY_SIZE has been exceeded, as with VAXC/VAX).
 #
-#DEFAULT_CACHE_SIZE:10
-#DEFAULT_VIRTUAL_MEMORY_SIZE:512000
+DEFAULT_CACHE_SIZE:512
+DEFAULT_VIRTUAL_MEMORY_SIZE:25165824
 
 .h2 SOURCE_CACHE
 # SOURCE_CACHE sets the source caching behavior for Lynx:
@@ -864,7 +862,7 @@ DEFAULT_INDEX_FILE:http://scout.wisc.edu
 # NONE is the default; the document source is not cached, and is reloaded
 #   from the network when needed.
 #
-#SOURCE_CACHE:NONE
+SOURCE_CACHE:MEMORY
 
 .h2 SOURCE_CACHE_FOR_ABORTED
 # This setting controls what will happen with cached source for the document
@@ -873,7 +871,7 @@ DEFAULT_INDEX_FILE:http://scout.wisc.edu
 # be preserved (and used as cache), if set to DROP lynx will drop the
 # source cache for that document (i.e. only completely downloaded documents
 # will be cached in that case).
-#SOURCE_CACHE_FOR_ABORTED:DROP
+SOURCE_CACHE_FOR_ABORTED:KEEP
 
 .h2 ALWAYS_RESUBMIT_POSTS
 # If ALWAYS_RESUBMIT_POSTS is set TRUE, Lynx always will resubmit forms
@@ -906,7 +904,7 @@ DEFAULT_INDEX_FILE:http://scout.wisc.edu
 # same image is indicated in the HTML markup.  The compilation default is
 # FALSE, such that a link with "[ISMAP]" as the link name, followed by a
 # hyphen, will be prepended to the ALT string or "[USEMAP]" pseudo-ALT for
-# accessing Lynx's text-based rendition of the client-side map (based on
+# accessing Lynx' text-based rendition of the client-side map (based on
 # the content of the associated MAP element).  If the "[ISMAP]" link is
 # activated, Lynx will send a 0,0 coordinate pair to the server, which
 # Lynx-friendly sites can map to a for-text-client document, homologous
@@ -925,7 +923,7 @@ DEFAULT_INDEX_FILE:http://scout.wisc.edu
 # The compilation default is to use the current document's URL in all
 # cases (i.e., assume the MAP is present below, if it wasn't present
 # above the point in the HTML stream where the USEMAP attribute was
-# detected).  Lynx's present "single pass" rendering engine precludes
+# detected).  Lynx' present "single pass" rendering engine precludes
 # checking below before making the decision on how to resolve a USEMAP
 # reference consisting solely of a fragment.
 #
@@ -943,7 +941,7 @@ DEFAULT_INDEX_FILE:http://scout.wisc.edu
 #SEEK_FRAG_AREA_IN_CUR:TRUE
 
 .h1 CGI scripts
-# These settings control Lynx's ability to execute various types of scripts.
+# These settings control Lynx' ability to execute various types of scripts.
 
 .h2 LOCAL_EXECUTION_LINKS_ALWAYS_ON
 .h2 LOCAL_EXECUTION_LINKS_ON_BUT_NOT_REMOTE
@@ -955,7 +953,7 @@ DEFAULT_INDEX_FILE:http://scout.wisc.edu
 # file for more detail on enabling execution links and scripts.
 #
 # If you have enabled execution links or scripts the following
-# two variables control Lynx's action when an execution link
+# two variables control Lynx' action when an execution link
 # or script is encountered.
 #
 # If LOCAL_EXECUTION_LINKS_ALWAYS_ON is set to TRUE any execution
@@ -1102,7 +1100,7 @@ DEFAULT_INDEX_FILE:http://scout.wisc.edu
 # default or that defined here can be toggled via the -force_secure
 # command line switch.
 #
-#FORCE_SSL_COOKIES_SECURE:FALSE
+FORCE_SSL_COOKIES_SECURE:TRUE
 
 .h1 Internal Behavior
 
@@ -1242,7 +1240,7 @@ DEFAULT_INDEX_FILE:http://scout.wisc.edu
 #SCROLLBAR:FALSE
 
 .h2 SCROLLBAR_ARROW
-# If SCROLLBAR_ARROW is set TRUE, Lynx's scrollbar will have arrows at the
+# If SCROLLBAR_ARROW is set TRUE, Lynx' scrollbar will have arrows at the
 # ends.  With mouse enabled, the arrows are clickable, and scroll the window by
 # 2 lines.  The appearance of the scrollbar arrows can be changed from LYNX_LSS
 # file:  define attributes scroll.arrow, scroll.noarrow (for enabled-arrows,
@@ -1281,7 +1279,7 @@ DEFAULT_INDEX_FILE:http://scout.wisc.edu
 # line break.  Note that the valid way to insert extra blank lines in HTML
 # is via a PRE block with only newlines in the block.
 #
-#COLLAPSE_BR_TAGS:TRUE
+COLLAPSE_BR_TAGS:FALSE
 
 .h2 TAGSOUP
 # If TAGSOUP is set, Lynx uses the "Tag Soup DTD" rather than "SortaSGML".
@@ -1379,7 +1377,7 @@ DEFAULT_INDEX_FILE:http://scout.wisc.edu
 # The default is FALSE, so that the feature needs to be enabled here
 # explicitly if you want it.
 #
-#PERSISTENT_COOKIES:FALSE
+PERSISTENT_COOKIES:TRUE
 
 .h2 COOKIE_FILE
 # COOKIE_FILE is the default file from which persistent cookies are read
@@ -1387,7 +1385,7 @@ DEFAULT_INDEX_FILE:http://scout.wisc.edu
 # USE_PERSISTENT_COOKIES and the PERSISTENT_COOKIES option is enabled.
 # The cookie file can also be specified in .lynxrc or on the command line.
 #
-#COOKIE_FILE:~/.lynx_cookies
+#COOKIE_FILE:~/.etc/cookies.txt
 
 .h2 COOKIE_SAVE_FILE
 # COOKIE_SAVE_FILE is the default file in which persistent cookies are
@@ -1399,7 +1397,7 @@ DEFAULT_INDEX_FILE:http://scout.wisc.edu
 # COOKIE_FILE if it is not set.  With a non-interactive Lynx session (e.g.,
 # -dump), cookies will only be saved to file if COOKIE_SAVE_FILE is set.
 #
-#COOKIE_SAVE_FILE:~/.lynx_cookies
+#COOKIE_SAVE_FILE:~/.etc/cookies.txt
 
 .h1 Mail-related
 
@@ -1544,7 +1542,7 @@ DEFAULT_INDEX_FILE:http://scout.wisc.edu
 # It will be prepended by the user's home directory.
 # NOTE that a file ending in .html or other suffix mapped to text/html
 # should be used to ensure its treatment as HTML.  The built-in default
-# is lynx_bookmarks.html.  On both Unix and VMS, if a subdirectory off of
+# is ./.etc/bookmark.htm.  On both Unix and VMS, if a subdirectory off of
 # the HOME directory is desired, the path should begin with "./" (e.g.,
 # ./BM/lynx_bookmarks.html), but the subdirectory must already exist.
 # Lynx will create the bookmark file, if it does not already exist, on
@@ -1556,7 +1554,7 @@ DEFAULT_INDEX_FILE:http://scout.wisc.edu
 # (see below), via the 'o'ptions menu, and can save those definitions
 # in the .lynxrc file.
 #
-#DEFAULT_BOOKMARK_FILE:lynx_bookmarks.html
+#DEFAULT_BOOKMARK_FILE:./.etc/bookmark.htm
 
 .h2 MULTI_BOOKMARK_SUPPORT
 # If MULTI_BOOKMARK_SUPPORT is set TRUE, and BLOCK_MULTI_BOOKMARKS (see
@@ -1634,6 +1632,7 @@ DEFAULT_INDEX_FILE:http://scout.wisc.edu
 #POSITIONABLE_EDITOR:jove
 #POSITIONABLE_EDITOR:jpico
 #POSITIONABLE_EDITOR:jstar
+#POSITIONABLE_EDITOR:jupp
 #POSITIONABLE_EDITOR:nano
 #POSITIONABLE_EDITOR:pico
 #POSITIONABLE_EDITOR:rjoe
@@ -1815,6 +1814,10 @@ DEFAULT_INDEX_FILE:http://scout.wisc.edu
 .ex
 #DOWNLOADER:Save OS/390 binary file: iconv -f IBM-1047 -t ISO8859-1 %s >%s:FALSE
 
+# These are configured by The MirOS Project:
+DOWNLOADER:View in less:less -f '%s':FALSE
+DOWNLOADER:Spawn editor:"${VISUAL-${EDITOR-ed}}" '%s':FALSE
+
 .h1 Interaction
 
 .h2 NO_DOT_FILES
@@ -1946,7 +1949,7 @@ DEFAULT_INDEX_FILE:http://scout.wisc.edu
 # prefer the more conservative action of requiring an explicit Y or y to
 # confirm.  The default defined here will override that in userdefs.h.
 #
-#QUIT_DEFAULT_YES:TRUE
+QUIT_DEFAULT_YES:FALSE
 
 .h1 HTML Parsing
 
@@ -2271,7 +2274,7 @@ MINIMAL_COMMENTS:TRUE
 # XLOADIMAGE_COMMAND definition.
 # If you use xli, you may want to add the -quiet flag.
 #
-#XLOADIMAGE_COMMAND:xli %s &
+#XLOADIMAGE_COMMAND:xloadimage -quiet %s
 
 .h2 VIEWER
 # MIME types may be assigned to external viewers using
@@ -2352,11 +2355,11 @@ MINIMAL_COMMENTS:TRUE
 # Accept.
 #
 # Values for this option are keywords:
-#	INTERNAL	lynx's built-in types for internal conversions
+#	INTERNAL	lynx' built-in types for internal conversions
 #	CONFIGFILE	adds lynx.cfg
 #	USER		adds PERSONAL_EXTENSION_MAP settings
 #	SYSTEM		adds GLOBAL_EXTENSION_MAP  settings
-#	ALL		adds lynx's built-in types for external conversions
+#	ALL		adds lynx' built-in types for external conversions
 #
 #PREFERRED_MEDIA_TYPES:internal
 
@@ -3217,7 +3220,7 @@ COLOR:6:brightred:black
 # otherwise. Set this setting to TRUE if you prefer numbered links, but wish
 # to get valid HTML source when printing or mailing when in psrc view.
 # Default is FALSE.
-#PRETTYSRC_VIEW_NO_ANCHOR_NUMBERING:FALSE
+PRETTYSRC_VIEW_NO_ANCHOR_NUMBERING:TRUE
 
 .h1 HTML Parsing
 
@@ -3252,19 +3255,19 @@ COLOR:6:brightred:black
 #	<p />
 #	<a />
 # When the option is false, lynx will not treat the tag as an ending.
-#XHTML_PARSING:FALSE
+XHTML_PARSING:TRUE
 
 .h1 Appearance
 
 .h2 JUSTIFY
 # JUSTIFY - Appearance
-# This option mirrors command-line option with same name.  Default is TRUE.  If
-# true, most of text (except headers and like this) will be justified.  This
-# has no influence on CJK text rendering.
+# This option mirrors command-line option with same name.  Default is TRUE.
+# If true, most of text (except headers and like this) will be justified.
+# This has no influence on CJK text rendering.
 #
 # This option is only available if Lynx was compiled with USE_JUSTIFY_ELTS.
 #
-#JUSTIFY:FALSE
+JUSTIFY:TRUE
 
 .h2 JUSTIFY_MAX_VOID_PERCENT
 # JUSTIFY_MAX_VOID_PERCENT - Appearance
@@ -3291,7 +3294,7 @@ COLOR:6:brightred:black
 # explicit activation can also be requested with the -tna command line
 # option.
 #
-#TEXTFIELDS_NEED_ACTIVATION:FALSE
+TEXTFIELDS_NEED_ACTIVATION:TRUE
 
 .h2 LEFTARROW_IN_TEXTFIELD_PROMPT
 # LEFTARROW_IN_TEXTFIELD_PROMPT
@@ -3301,7 +3304,7 @@ COLOR:6:brightred:black
 # only if the contents of the fields have been changed since entering it.
 # If set to TRUE, the confirmation prompt is always issued.
 #
-#LEFTARROW_IN_TEXTFIELD_PROMPT:FALSE
+LEFTARROW_IN_TEXTFIELD_PROMPT:TRUE
 
 .h1 Timeouts
 
@@ -3340,8 +3343,9 @@ COLOR:6:brightred:black
 # one lynx.cfg setting.
 .nf
 #ENABLE_LYNXRC:ACCEPT_ALL_COOKIES:ON
-#ENABLE_LYNXRC:ASSUME_CHARSET:OFF
+ENABLE_LYNXRC:ASSUME_CHARSET:ON
 #ENABLE_LYNXRC:AUTO_SESSION:OFF
+ENABLE_LYNXRC:BAD_HTML:ON
 #ENABLE_LYNXRC:BOOKMARK_FILE:ON
 #ENABLE_LYNXRC:CASE_SENSITIVE_SEARCHING:ON
 #ENABLE_LYNXRC:CHARACTER_SET:ON
@@ -3356,44 +3360,43 @@ COLOR:6:brightred:black
 #ENABLE_LYNXRC:EMACS_KEYS:ON
 #ENABLE_LYNXRC:FILE_EDITOR:ON
 #ENABLE_LYNXRC:FILE_SORTING_METHOD:ON
-#ENABLE_LYNXRC:FORCE_COOKIE_PROMPT:OFF
-#ENABLE_LYNXRC:FORCE_SSL_PROMPT:OFF
-#ENABLE_LYNXRC:FTP_PASSIVE:OFF
-#ENABLE_LYNXRC:HTML5_CHARSETS:OFF
+ENABLE_LYNXRC:FORCE_COOKIE_PROMPT:ON
+ENABLE_LYNXRC:FORCE_SSL_PROMPT:ON
+ENABLE_LYNXRC:FTP_PASSIVE:ON
+ENABLE_LYNXRC:HTML5_CHARSETS:ON
 #ENABLE_LYNXRC:KBLAYOUT:ON
 #ENABLE_LYNXRC:KEYPAD_MODE:ON
 #ENABLE_LYNXRC:LINEEDIT_MODE:ON
 #ENABLE_LYNXRC:LOCALE_CHARSET:ON
-#ENABLE_LYNXRC:MAKE_LINKS_FOR_ALL_IMAGES:OFF
-#ENABLE_LYNXRC:MAKE_PSEUDO_ALTS_FOR_INLINES:OFF
+ENABLE_LYNXRC:MAKE_LINKS_FOR_ALL_IMAGES:ON
+ENABLE_LYNXRC:MAKE_PSEUDO_ALTS_FOR_INLINES:ON
 #ENABLE_LYNXRC:MULTI_BOOKMARK:ON
-#ENABLE_LYNXRC:NO_PAUSE:OFF
+ENABLE_LYNXRC:NO_PAUSE:ON
 #ENABLE_LYNXRC:PERSONAL_MAIL_ADDRESS:ON
 #ENABLE_LYNXRC:PREFERRED_CHARSET:ON
-#ENABLE_LYNXRC:PREFERRED_ENCODING:OFF
+ENABLE_LYNXRC:PREFERRED_ENCODING:ON
 #ENABLE_LYNXRC:PREFERRED_LANGUAGE:ON
-#ENABLE_LYNXRC:PREFERRED_MEDIA_TYPES:OFF
-#ENABLE_LYNXRC:RAW_MODE:OFF
+ENABLE_LYNXRC:PREFERRED_MEDIA_TYPES:ON
+ENABLE_LYNXRC:RAW_MODE:ON
 #ENABLE_LYNXRC:RUN_ALL_EXECUTION_LINKS:ON
 #ENABLE_LYNXRC:RUN_EXECUTION_LINKS_ON_LOCAL_FILES:ON
-#ENABLE_LYNXRC:SCROLLBAR:OFF
+ENABLE_LYNXRC:SCROLLBAR:ON
 #ENABLE_LYNXRC:SELECT_POPUPS:ON
 #ENABLE_LYNXRC:SEND_USERAGENT:OFF
 #ENABLE_LYNXRC:SESSION_FILE:OFF
-#ENABLE_LYNXRC:SET_COOKIES:OFF
+ENABLE_LYNXRC:SET_COOKIES:ON
 #ENABLE_LYNXRC:SHOW_COLOR:ON
 #ENABLE_LYNXRC:SHOW_CURSOR:ON
 #ENABLE_LYNXRC:SHOW_DOTFILES:ON
-#ENABLE_LYNXRC:SHOW_KB_RATE:OFF
+ENABLE_LYNXRC:SHOW_KB_RATE:ON
 #ENABLE_LYNXRC:SUB_BOOKMARKS:ON
-#ENABLE_LYNXRC:TAGSOUP:OFF
-#ENABLE_LYNXRC:UNDERLINE_LINKS:OFF
-#ENABLE_LYNXRC:USER_MODE:ON
-#ENABLE_LYNXRC:SEND_USERAGENT:OFF
+ENABLE_LYNXRC:TAGSOUP:ON
+ENABLE_LYNXRC:UNDERLINE_LINKS:ON
 #ENABLE_LYNXRC:USERAGENT:OFF
+#ENABLE_LYNXRC:USER_MODE:ON
 #ENABLE_LYNXRC:VERBOSE_IMAGES:ON
-#ENABLE_LYNXRC:VI_KEYS:ON
 #ENABLE_LYNXRC:VISITED_LINKS:ON
+#ENABLE_LYNXRC:VI_KEYS:ON
 .fi
 
 .h1 External Programs
